components/core/tests/tests_User_endpoint_tests.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import requests
+import logging as logger
+from flask import g
+
+headers = {
+    'Host': 'localhost:5000',
+    'Accept': 'application/json, text/plain, */*',
+    'Accept-Language': 'en-US,en;q=0.5',
+    'Accept-Encoding': 'gzip, deflate',
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'Origin': 'http://localhost:3000',
+}
+user_data = {
```

Comment:
Will we need to load these `passwords/user_names` from a config?

Reply:
Right now these are just present in this file itself in future I will create a config to store this data.

Reply:
OK. 😀